simplq/src/store/asyncActions/getQueueInfo.js

```diff
@@ -12,13 +12,9 @@ const typePrefix = 'getQueueInfo/action';
 const useGetQueueInfo = () => {
   const makeAuthedRequest = useMakeAuthedRequest();
 
-  const getQueueInfo = createAsyncThunk(typePrefix, async ({ queueId }) => {
-    const authedRequest = makeAuthedRequest(RequestFactory.getQueueInfo(queueId));
-    const response = await authedRequest;
-    return response;
-  });
-
-  return getQueueInfo;
+  return createAsyncThunk(typePrefix, async ({ queueId }) =>
+    makeAuthedRequest(RequestFactory.getQueueInfo(queueId))
+  );
```

Comment:
Removing the return response here is a problem I believe. If you look at the queueInfoSlice script in the store directory, you'll see that we have a reducer that stores the response from this thunk into the state which is displayed in the admin or join pages.
```
    [getQueueInfo.fulfilled]: (state, action) => {
      return action.payload;
    },
```
So this change would need to be reverted.

Reply:
Apart from returning the API response body, let's revert the changes in this file and the following asyncActions files. The reason is that even though directly returning the thunks without naming them is arguably a better practice according to sonarcloud, we had decided before to leave the asyncActions thunks the way they were because we felt that it was a little more readable. It's a subjective choice, but we're going to stick to it. For the remaining asyncActions files, we had removed the related warnings from sonarcloud, but I guess we missed these since they were recently modified. Sorry about that. @daltonfury42 could you please remove the unnecessary smells from sonarcloud?

Reply:
There are two separate things here:
A:

> Removing the return response here is a problem I believe. 

@maaverik he is using lambda without braces, so the expression becomes the return value:
```
(a, b) => a + b
``` 
vs 
```
(a, b) => {
  return a + b;
}
```
where the former is preferable. 

B: In the actions, we have decided to prefer using this form for readability:
```
  const getUserQueues = createAsyncThunk(typePrefix, async () => {
    ...
  });

  return getUserQueues;
```

over returning directly like this:
```
  return createAsyncThunk(typePrefix, async () => {
    ...
  });
```

So combining these two things, what we should do here is:

```
const useGetQueueInfo = () => {
  const makeAuthedRequest = useMakeAuthedRequest();
  
  const getQueueInfo = createAsyncThunk(typePrefix, async ({ queueId }) =>
    makeAuthedRequest(RequestFactory.getQueueInfo(queueId))
  );

  return getQueueInfo; 
}
```

> could you please remove the unnecessary smell warnings from sonarcloud?

On it.

Reply:
> @maaverik he is using lambda without braces, so the expression becomes the return value:

Ah, my bad, didn't pay attention to the braces.